### PR TITLE
add the interface for origin-destination CSV

### PR DESCRIPTION
**Pull request by shenriotpro**
# Description

add the interface for origin-destination CSV

# Issues

close https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/issues/268

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I've read the [Contribution Guidelines](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/main/CONTRIBUTING.md)
* [ ] I've added tests for changes or features I've introduced
* [x] I documented any high-level concepts I'm introducing in `documentation/`
* [ ] CI is currently green and this is ready for review

I could use help for:

- What kind of tests would you like?
- What kind of documentation would you like?
- Should we hide the button while the feature is WIP?
- Is this 5-column format OK, rather than 3 matrices?
- What should be the German translations?
- I tried prettier but it seems the current code is not compliant.

### CODE REVIEW

**Comment by louisgreiner** [comment]
To answer to your questions:
 > What kind of documentation would you like?
We could use what has been written in [the feature request](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/issues/126) for some context (business related, Adrian or I could write that), and [this issue](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/issues/199) for the technical part (algorithms, assumptions, what is the aim of this first version). It would be great to write this documentation here : [netzgrafik-editor-frontend/documentation](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/tree/main/documentation).
> Should we hide the button while the feature is WIP?
Yes, I left a comment on that
> Is this 5-column format OK, rather than 3 matrices?
I would say yes, since we just want to confirm the feasibility of the feature on larger networks (so it has to be readable at least)
> What should be the German translations?
We can wait for Adrian to deal with that in a few days, and stick to english for now.

For the questions left, I prefer let @aiAdrian answer when he's back

**Review by louisgreiner — COMMENTED** [review]
left a few comments, but overall looking good to me

**Review by louisgreiner — APPROVED** [review]
Lgtm, thank you :+1:

File: documentation/ORIGIN_DESTINATION_MATRIX.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ If some nodes are selected, then the Origin Destination Matrix will only show re
 
 The Origin Destination Matrix will only show results between visible nodes (but other nodes may be used to compute paths).
 
-If some trainruns are selected, then the Origin Destination Matrix will only use these trainruns to compute paths.
\ No newline at end of file
+The Origin Destination Matrix will only use visible traintuns to compute paths.
```
Comment (louisgreiner) [inline documentation/ORIGIN_DESTINATION_MATRIX.md:13]:
```suggestion
The Origin Destination Matrix will only use visible trainruns to compute paths.
```

**Review by louisgreiner — APPROVED** [review]

**Review by aiAdrian — APPROVED** [review]

**Pull request merged**